lib/config.ts

```diff
@@ -14,7 +19,22 @@ if (!GRAPHQL_HOSTNAME) {
   }
 }
 
+if (!GRAPHQL_HOSTNAME_INTERNAL) {
```

Comment:
Why this logic? If its not set it should exit with error or?
This is only relevant for server side right?

Reply:
for the same reason the code block above is here, we have to have multiple dns set up for bbw for the time being.

Reply:
my bad. actually yes this is only server side so this is irrelevant